README.md

```diff
@@ -20,22 +20,33 @@ provided by [nim-markdown]. For a quick reference of supported syntax see the [c
 
 ### Status
 
-nimibook currently provides minimal functionality to create a book, notably missing is
-a command line interface to automate common task (e.g. initialization of a new nimibook).
+nimibook currently provides minimal functionality to create a book and support a local CLI mode.
 
 ## Installation
 
 To be able to use nimibook, currently the only way is to recreate the same content of the repository
```

Comment:
I think we can update the installation instruction, right? it should be: 1) install nimibook 2) create a simple nim file (`nbook.nim`) with content (`import nimibook; nbBookTasks`) and run `nim r nbook init` in the folder where you want the book. Should we also rename `nbBookTasks` to `nimibookCli`?

Reply:
Seems reasonable. I'll modify it after lunch :)